HARK/simulation/simulator.py

```diff
@@ -0,0 +1,3171 @@
+        raise ValueError("Statement line was not any valid type!")
+
+    # Now make and return an appropriate event for that type
+    if event_type is DynamicEvent:
+        event_maker = make_new_dynamic
+    if event_type is RandomEvent:
+        event_maker = make_new_random
+    if event_type is RandomIndexedEvent:
+        event_maker = make_new_random_indexed
+    if event_type is MarkovEvent:
+        event_maker = make_new_markov
+    if event_type is EvaluationEvent:
+        event_maker = make_new_evaluation
+
+    new_event, names_used = event_maker(statement, info)
+    return new_event, names_used
+
+
+def make_new_dynamic(statement, info):
+    """
+    Construct a new instance of DynamicEvent based on the given model statement
+    line and a blank dictionary of parameters. The statement should already be
+    verified to be a valid dynamic statement: it has an = but no ~ or @.
+
+    Parameters
+    ----------
+    statement : str
+        One line dynamics statement, which will be turned into a DynamicEvent.
+    info : dict
+        Empty dictionary of available information.
+
+    Returns
+    -------
+    new_dynamic : DynamicEvent
+        A new dynamic event with values and information missing, but structure set.
+    names_used : [str]
+        List of names of objects used in this expression.
+    """
+    # Cut the statement up into its LHS, RHS, and description
+    lhs, rhs, description = parse_line_for_parts(statement, "=")
+
+    # Parse the LHS (assignment) to get assigned variables
+    assigns = parse_assignment(lhs)
+
+    # Parse the RHS (dynamic statement) to extract object names used
+    obj_names, is_indexed = extract_var_names_from_expr(rhs)
+
+    # Allocate each variable to needed dynamic variables or parameters
+    needs = []
+    parameters = {}
+    for j in range(len(obj_names)):
+        var = obj_names[j]
+        if var not in info.keys():
+            raise ValueError(
+                var + " is used in a dynamic expression, but does not (yet) exist!"
+            )
+        val = info[var]
+        if type(val) is NullFunc:
+            raise ValueError(
+                var + " is used in a dynamic expression, but it's a function!"
+            )
+        if type(val) is Distribution:
+            raise ValueError(
+                var + " is used in a dynamic expression, but it's a distribution!"
+            )
+        if val is None:
+            parameters[var] = None
+        else:
+            needs.append(var)
+
+    # Declare a SymPy symbol for each variable used; these are temporary
+    _args = []
+    for j in range(len(obj_names)):
+        _var = obj_names[j]
+        if is_indexed[j]:
+            exec(_var + " = IndexedBase('" + _var + "')")
+        else:
+            exec(_var + " = symbols('" + _var + "')")
+        _args.append(eval(_var))
+
+    # Make a SymPy expression, then lambdify it
+    sympy_expr = eval(rhs)
+    expr = lambdify(_args, sympy_expr)
```

Comment:
```suggestion
        _args.append(symbols(_var))

    # Make a SymPy expression, then lambdify it
    sympy_expr = symbols(rhs)
    expr = lambdify(_args, sympy_expr)
```

This should fix the issue with py3.13